package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "logform",
-  "version": "1.2.2",
+  "version": "1.3.0",
```

Comment:
Please do not include version bumps in PRs. As a maintainer team we may merge multiple features into a future minor version bump.

Reply:
No problem.  This makes total sense, I'm just used to doing it haha.  I'll edit accordingly.